src/hathor.c

```diff
@@ -11,33 +11,44 @@
 #include <cx.h>
 #include "hathor.h"
 
+#define B58_MAX_INPUT_SIZE 120
+
+// All keys that we derive start with path 44'/280'/0'
+const uint32_t htr_bip44[] = { 44 | 0x80000000, 280 | 0x80000000, 0 | 0x80000000 };
```

Comment:
Is the bit `0x80000000` for hardened keys? Maybe we should leave a comment about it.

Reply:
Yes, it's specified in BIP32.

Reply:
I didn't know that. Thanks!

Even though, maybe we should leave a comment with a link to BIP32 specification. What do you think?

Reply:
Added the comment.